### DISCUSSION TITLE
BD已经不允许创建apps目录了

### DISCUSSION THREAD

**Discussion by deepcomp2023** [houtianze/bypy#633] [General]
为了规避这个问题，我只好改了下源码再安装包。

但是各位亲，bypy命令如何更新授权？是在auth.json里面，还是在bypy.json里面？

**Comment by Zhijiang-Li1111** [comment]
看起来像是老的接口废弃了？我用老接口一直返回31064错误，但是用新接口成功了https://pan.baidu.com/union/doc/nksg0sat9
BTW，apps就是我的应用数据目录

**Comment by houtianze** [comment]
哦，这样啊，我看看，谢谢提供信息。On 11 Jan 2023, at 10:40 PM, Zhijiang-Li1111 ***@***.***> wrote:﻿
看起来像是老的接口废弃了？我用老接口一直返回31064错误，但是用新接口成功了https://pan.baidu.com/union/doc/nksg0sat9
BTW，apps就是我的应用数据目录

**Comment by houtianze** [comment]
@SpotInspect `bypy --clean`后就可以重新授权（`bypy.json`)

**Comment by houtianze** [comment]
看了一下，新的API貌似并不合旧的兼容。。🤦‍♂️

**Reply by Zhijiang-Li1111** [reply]
是的，应该要适配新的接口了，直接改域名不好使，会报参数错误

**Comment by houtianze** [comment]
还是没太懂，现在用旧的api的问题是什么？

On Thu, 12 Jan 2023 at 11:15 AM, Zhijiang-Li1111 ***@***.***>
wrote:

> 是的，应该要适配新的接口了，直接改域名不好使，会报参数错误
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/houtianze/bypy/discussions/633#discussioncomment-4663090>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAYFZXQMH5LS3TQSEA3PQDWR5ZMFANCNFSM6AAAAAATMUVANA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Regards,
Tianze

(Typed on phone, please excuse any typos, thank you)

**Reply by Zhijiang-Li1111** [reply]
返回31064错误

**Comment by houtianze** [comment]
我i试了一下`ls`没问题。是什么api返回31064？有log分享一下吗？
